src/torch_tomogram/tomogram.py

```diff
@@ -27,13 +34,95 @@ def __init__(
         self.sample_translations = torch.as_tensor(
             sample_translations, device=device
         ).float()
+        self.pixel_spacing = pixel_spacing
         self.device = device
         self._pad_factor = 2.0
+    
+    @property
+    def sample_translations_px(self) -> torch.Tensor:
+        """Get sample translations in pixels.
+        
+        If pixel_spacing was provided, converts from Angstroms to pixels.
+        Otherwise, returns the translations as-is (already in pixels).
+        """
+        if self.pixel_spacing is not None:
+            return self.sample_translations / self.pixel_spacing
+        return self.sample_translations
+
+    @classmethod
+    def from_aretomo_aln(
+        cls,
+        aln_path: Path | str,
+        tilt_stack_path: Path | str,
+        pixel_spacing: float | None = None,
+        device: torch.device | str = "cpu",
+    ) -> "Tomogram":
+        """Initialize Tomogram from AreTomo alignment file and tilt stack."""
+
+        aln_path = Path(aln_path)
+        tilt_stack_path = Path(tilt_stack_path)
+        df = alnfile.read(aln_path)
+        corrected_shifts_xy = df[["tx", "ty"]].to_numpy()
+        corrected_shifts_yx = corrected_shifts_xy[:, ::-1].copy()
+        
+        # Convert shifts to Angstroms if pixel_spacing provided
+        if pixel_spacing is not None:
+            corrected_shifts_yx = corrected_shifts_yx * pixel_spacing
+        
+        tilt_stack_full = mrcfile.read(tilt_stack_path)
+        included_indices = df["sec"].values - 1 
+        tilt_stack = tilt_stack_full[included_indices]
+        tilt_stack = tilt_stack.astype(np.float32)
+        tilt_stack -= np.mean(tilt_stack, axis=(-2, -1), keepdims=True)
+        tilt_stack /= np.std(tilt_stack, axis=(-2, -1), keepdims=True)
+        return cls(
+            images=tilt_stack,
+            tilt_angles=df["tilt"].to_numpy(),
+            tilt_axis_angle=df["rot"].to_numpy(),
+            sample_translations=corrected_shifts_yx,
+            pixel_spacing=pixel_spacing,
+            device=device,
+        )
+
+    @classmethod
+    def from_etomo_directory(
+        cls,
+        etomo_dir: Path | str,
+        pixel_spacing: float | None = None,
+        device: torch.device | str = "cpu",
+    ) -> "Tomogram":
+        """Initialize Tomogram from ETOMO directory."""
+
+        etomo_dir = Path(etomo_dir)
```

Comment:
same q as above, can this be removed?
```suggestion

        etomo_dir = Path(etomo_dir)
```